### PR TITLE
[TEST] bump golang.org/x/net v0.0.0-20220927171203-f486391704dc

### PR DESCRIPTION
**Pull request by aojea**
/kind bug
/kind cleanup

#### What this PR does / why we need it:

Update golang.org/x/net dependency

```release-note
NONE
```

### CODE REVIEW

**Comment by aojea** [comment]
/hold

debug https://github.com/kubernetes/kubernetes/pull/112764#issuecomment-1260550507

File: staging/src/k8s.io/api/go.mod [hunk 1]
```diff
@@ -35,6 +35,7 @@ require (
 )
 
 replace (
+	golang.org/x/net => golang.org/x/net v0.0.0-20220722155237-a158d28d115b
```
Comment (aojea) [inline staging/src/k8s.io/api/go.mod:38]: I'm doing something wrong

Comment (brianpursley) [reply staging/src/k8s.io/api/go.mod:38]: Did you mean `v0.0.0-20220927171203-f486391704dc` here instead?

Comment (brianpursley) [reply staging/src/k8s.io/api/go.mod:38]: Was that something you added or was it done my pin-dependency.sh?

I seem to recall having to run through the series of steps multiple times until verify returned nothing:

1. Use hack/pin-dependency.sh to switch to the preferred version for each module
2. Run hack/update-vendor.sh to rebuild the vendor directory
3. Run hack/lint-dependencies.sh to verify no additional changes are required

Apologies if you already know this.

Comment (dims) [reply staging/src/k8s.io/api/go.mod:38]: CI job is telling the same thing :)

```
Check all unwanted dependencies and update its status.
These modules are pinned to versions different than the minimal preferred version.
That means that without replace directives, a different version would be selected,
which breaks consumers of our published modules.
1. Use hack/pin-dependency.sh to switch to the preferred version for each module
2. Run hack/update-vendor.sh to rebuild the vendor directory
3. Run hack/lint-dependencies.sh to verify no additional changes are required

golang.org/x/net
    pinned:    v0.0.0-20220722155237-a158d28d115b
    preferred: v0.0.0-20220927171203-f486391704dc
    hack/pin-dependency.sh golang.org/x/net v0.0.0-20220927171203-f486391704dc
All pinned versions of checked dependencies match their preferred version.
```

Comment (aojea) [reply staging/src/k8s.io/api/go.mod:38]: ic, so I have to pin it, make sense

**Comment by aojea** [comment]
> #112693 Is it a duplicate?

it is a duplicate 🙃

**Pull request closed without merging**